pkg/api/vm/schema.go

```diff
@@ -15,22 +15,31 @@ const (
 )
 
 func RegisterSchema(scaled *config.Scaled, server *server.Server) {
+	server.BaseSchemas.MustImportAndCustomize(EjectCdRomActionInput{}, nil)
```

Comment:
can we please add a comment to describe how the base scheme and resource action is integrated, e.g,

```
// base scheme is used to customize the input data of resource actions, the action input name should start with lower case.
```